ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/service/ViewServiceImpl.java

```diff
@@ -30,12 +36,29 @@ public ViewServiceImpl(ViewRepository viewRepository, ApplicationEventPublisher
 	@Override
 	public void addView(ViewSpecification viewSpecification) {
 		Optional<ViewSpecification> view = viewRepository.getViewByViewName(viewSpecification.getName());
-		if (view.isEmpty()) {
-			viewRepository.saveView(viewSpecification);
-			eventPublisher.publishEvent(new ViewAddedEvent(viewSpecification));
-		} else {
+		if (view.isPresent()) {
 			throw new DuplicateViewException(viewSpecification.getName());
 		}
+
+		viewRepository.saveView(viewSpecification);
+		eventPublisher.publishEvent(new ViewAddedEvent(viewSpecification));
+	}
+
+	@Override
+	public void addDefaultView(String collectionName) {
+		ViewName defaultViewName = new ViewName(collectionName, DEFAULT_VIEW_NAME);
+		Optional<ViewSpecification> view = viewRepository.getViewByViewName(defaultViewName);
+		if (view.isPresent()) {
+			throw new DuplicateViewException(defaultViewName);
+		}
+
+		FragmentationConfig fragmentation = new FragmentationConfig();
+		fragmentation.setName(DEFAULT_VIEW_FRAGMENTATION_STRATEGY);
+		fragmentation.setConfig(DEFAULT_VIEW_FRAGMENTATION_PROPERTIES);
+
+		ViewSpecification defaultView = new ViewSpecification(defaultViewName, List.of(), List.of(fragmentation));
+		viewRepository.saveView(defaultView);
+		eventPublisher.publishEvent(new ViewAddedEvent(defaultView));
```

Comment:
implementation is ok, but maybe it's an idea to create a ViewSpecification based on the collection name and the default parameters in the EventStreamServiceImpl and use addView.
That would also avoid the duplication of `view.isPresent`, `viewRepository.saveView` and `publishEvent.`